homeassistant/components/device_tracker/fritz.py

```diff
@@ -0,0 +1,94 @@
+""" Supports scanning a FritzBox router"""
```

Comment:
Can you update the header comment to be similar to [luci](https://github.com/balloob/home-assistant/blob/dev/homeassistant/components/device_tracker/luci.py)?